pipelines/release-images-dev/tasks/build_final_release.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+mkdir /bosh-cache
+
+ROOT_DIR=$PWD
+
+pushd ci/bosh-releases/$RELEASE_NAME
```

Comment:
can you double quote the RELEASE_NAME, based on https://github.com/koalaman/shellcheck/wiki/SC2086